alex/corpustools/recording_splitter/recording_splitter.py

```diff
@@ -0,0 +1,356 @@
+from collections import deque
```

Comment:
do not put it into one one directory as a unique file. just put it into the corpustools